### PR TITLE
[ElasticSan] Update changelog for ElasticSan stable release

### PR DESCRIPTION
**Pull request by yifanz7**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by archerzz — APPROVED** [review]

File: sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-
-### Breaking Changes
-
-### Bugs Fixed
+- Supported snapshot and CMK for ElasticSan
+- Supported private endpoints for ElasticSan
+- Supported ElasticSan basic operations 
 
 ### Other Changes
+This is the first stable release of ElasticSan client library.
```
Comment (live1206) [inline sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md:11]: Since this is first GA, we need to verity if all SDK type names are fine.

Comment (yifanz7) [reply sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md:11]: Anything that we should do for the verification?

Comment (live1206) [reply sdk/elasticsan/Azure.ResourceManager.ElasticSan/CHANGELOG.md:11]: This will be done by SDK team member.

**Review by ArthurMa1978 — CHANGES_REQUESTED** [review]
Pending on https://github.com/Azure/azure-sdk-for-net/pull/41171

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**